app/templates/main/resource.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+<div class="reource block" style = "margin: 20px">
+<h1 class="ui header">Resource Name</h1>
+    <div class="ui horizontal label">Category</div>
+      Address 2304 Irving Street Philadelphia, PA <!-- This does not have an element put this in a div class that lines up with categories-->
+<h5 class="ui header">Info</h5>
+ <div class="text descriptor"><b>descriptor text</b> text text</div>
+    <div class="option descriptor"><b> option texts</b>

```

Comment:
no need for indent, the things under it before the `</div>` block should be indented one level from the `<div>` block